src/api-specification/contribution-controller/retrieving-contributions.md

```diff
@@ -20,11 +20,11 @@ The following parameters can be specified with the request to filter the retriev
 | Field | Value Type | Description |
 | :-: | --- | --- |
 | `size` | `integer($int32)` | The number of entries, starting with the latest, shown in the response. <br> Set to `50` by default. |
-| `from` and `to` | `string` | The timeframe that the response entries are filtered by (set as [Unix Epoch time](https://www.epochconverter.com/clock) in seconds). <br> The `from` parameter is set to `0` by default. |
+| `from` and `to` | `string` | The timeframe that the response entries are filtered by. Both are set as an exact time and date (ISO 8601: `YYYY-MM-DDTHH:MM:SSZ`). <br> If the `from` parameter is left unspecified, the results will be fetched since the establishment of the FIB network. |
```

Comment:
what about `to` being unspecified? in that case it'd search up to the latest record